tox.ini

```diff
@@ -72,7 +73,10 @@ passenv =
     {tests,functests}: PYTEST_ADDOPTS
     functests: BROKER_URL
 setenv =
+    PYTHONUNBUFFERED = 1
```

Comment:
This was being set in `hypothesis devserver`. I've set this for all envs not just dev, I think we just want it on all the time